src/vehicle_autodrive.cpp

```diff
@@ -722,7 +722,7 @@ cata::optional<navigation_step> vehicle::autodrive_controller::compute_next_step
     while( !data.path.empty() && data.path.back().pos != veh_pos ) {
         data.path.pop_back();
     }
-    if( !data.path.empty() && data.path.back().target_speed_tps > data.max_speed_tps) {
+    if( !data.path.empty() && data.path.back().target_speed_tps > data.max_speed_tps ) {
```

Comment:
I normally squash everything when I rebase (which is about once a week or when I need to pick up a bugfix or dependency). Squashing, like rebasing, requires a force-push and I'm afraid to do that often since I've see Github lose review comments before because of that.

Reply:
Fair enough.